src/Sylius/Bundle/AdminBundle/templates/promotion/grid/field/usage.html.twig

```diff
@@ -1,4 +1,4 @@
-{% if data.usageLimit is empty %}
+{% if not data.usageLimit %}
```

Comment:
This fix indeed resolves 500 error that is triggered, but it doesn't resolve the problem entirely, because after your change, when usage limit is set to 0, the list of promotions shows that there are infinitely many (∞), not 0 as it should. You should fix the progress bar, which is defined in line: 16 as it is a source of a problem here 🤔

Reply:
A usage limit of 0 makes no sense. The zero value must be invalid like the promotion coupon's usage limit.
Another option is usage 0 like `null`.

Reply:
Absolutely agree with you, my suggestion was about your attempt to fix the problem, which was even more misleading from my perspective. However, you are right, the best solution would be to solve it in the same way as with coupons, by making it impossible to set 0. Would you be willing to contribute with such a change? 😃

Reply:
Why not! But in other PR and after this PR are merged. I write the patch to solve quickly the bug.